discord/ext/commands/context.py

```diff
@@ -396,4 +397,22 @@ async def send_help(self, *args: Any) -> Any:
 
     @discord.utils.copy_doc(Message.reply)
     async def reply(self, content: Optional[str] = None, **kwargs: Any) -> Message:
-        return await self.message.reply(content, **kwargs)
+        msg = await self.message.reply(content, **kwargs)
+        if self._original_response_message is None:
+            self._original_response_message = msg
+        return msg
+
+    async def delete(self, *, delay: Optional[float] = None, reason: Optional[str] = None) -> None:
```

Comment:
I feel like this should be in BridgeContext, as it doesn't really make sense as a part of prefixed commands (there's no explicit response)

Reply:
That can definitely be done, just figured there could be use cases wherein a user replies via a regular Context object and may want to delete said response later on.

Reply:
That could be handled by the user storing the value of `reply`.

This just doesn't make much sense as there are a few different responses to a prefixed command, it could be a DM, reaction, reply, normal message, etc.